modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view.jsp

```diff
@@ -169,6 +169,19 @@ DLViewDisplayContext dlViewDisplayContext = new DLViewDisplayContext(dlAdminDisp
 						<liferay-ui:error exception="<%= FileEntryLockException.MustOwnLock.class %>" message="you-can-only-checkin-documents-you-have-checked-out-yourself" />
 						<liferay-ui:error key="externalServiceFailed" message="you-cannot-access-external-service-because-you-are-not-allowed-to-or-it-is-unavailable" />
 
+						<c:if test='<%= SessionErrors.contains(renderRequest, "googleDriveFileMissing") %>'>
+							<aui:script>
+								Liferay.Util.openToast({
+									message: '<liferay-ui:message key="the-google-drive-file-was-missing" />',
+									title: Liferay.Language.get('warning'),
+									toastProps: {
+										autoClose: 5000,
+									},
+									type: 'warning',
+								});
+							</aui:script>
+						</c:if>
+
```

Comment:
@ambrinchaudhary Isn't there a better way to open the toast? Shouldn't this be handled with a liferay-ui:error tag?

Reply:
I tried using `liferay-ui` and what is shown is the same as option 2 I mentioned in [Slack thread](https://liferay.slack.com/archives/CLD39UKM5/p1668571278941749). So that's why I opened a toast.
I would be happy to learn another way to do it.

Reply:
Yes, that is the correct way to open a toast message, as decided via slack. Looks ok to me.